packages/tablev2/Util.tsx

```diff
@@ -144,11 +144,15 @@ export type SorterFactory<T> = <B extends string, A extends Record<B, T>>(
   prop: B
 ) => Sorter<A>;
 
-const string: SorterFactory<string> = prop => (_, dir) => (a, b) =>
-  dir * a[prop].localeCompare(b[prop]);
+const string: SorterFactory<string | null | undefined> = prop => (_, dir) => (
+  a,
+  b
+) => dir * (a[prop] || "")!.localeCompare((b[prop] || "")!);
 
-const number: SorterFactory<number> = prop => (_, dir) => (a, b) =>
-  dir * (a[prop] - b[prop]);
+const number: SorterFactory<number | null | undefined> = prop => (_, dir) => (
```

Comment:
i'm not sure about this change.

i'd find it more useful if the `null`s and `undefined`s would be always last (like the `sortGroupsAlwaysOnTop`-thing from DCOS-UI's ServiceTable). What do you think?

Reply:
I agree with you. How else might we change these functions so they accept properties from `data` that could be null or undefined?